libwfssl/src/ssl.c

```diff
@@ -573,6 +573,17 @@ WF_OPENSSL(jlong, makeSSLContext)(JNIEnv *e, jobject o, jint protocol, jint mode
         if (!(protocol & SSL_PROTOCOL_TLSV1_2))
             ssl_methods.SSL_CTX_ctrl((c->ctx),SSL_CTRL_OPTIONS,(SSL_OP_NO_TLSv1_2),NULL);
     }
+
+    /* We need to disable TLSv1.3 if running on OpenSSL 1.1.0+ */
+#if OPENSSL_VERSION_NUMBER >= 0x10100000L
```

Comment:
What is the exact reason to disable TLSv1.3 completely here?

Reply:
TLSv1.3 is not supported at the moment. It brings too many changes that might require some rewriting to implement.

@fl4via, would it be safe to say that wildfly-openssl 1.0.x will be TLSv1.2 only and 1.1.x will support TLSv1.3?

Reply:
@iweiss probably, or maybe even 2.x, I'll have to define an implementation plan for this